mesa/space.py

```diff
@@ -21,8 +21,10 @@
 from __future__ import annotations
```

Comment:
Can you please update the space module (and can you update the pylint as well as we switched to ruff, if I am reading the documentation correctly you should just be able to remove line 15-17)

Reply:
https://github.com/projectmesa/mesa/pull/1898/commits/3e7bd7b6c016e6c9957969f00fd74d8c46f371c5